invenio_orcid/tasks.py

```diff
@@ -142,38 +139,36 @@ def send_to_orcid(sender, api=None):
                         api.update_record(author_orcid, token,
                                           'work', orcid_json, str(put_code))
                     current_app.logger.info(
-                        'Succersfully sent "{0}" to orcid.').format(
-                            sender.get(record_identifier))
+                        'Succersfully sent "{0}" to orcid.'.format(
+                            sender.get(record_identifier)))
                 except RequestException as e:
                     current_app.logger.info(
                         e.response.text, sender[record_identifier])
                     current_app.logger.info(
-                        'Failed to push "{0}" to orcid.').format(
-                            sender.get(record_identifier))
+                        'Failed to push "{0}" to orcid.'.format(
+                            sender.get(record_identifier)))
                     continue
         except (KeyError, AttributeError, TypeError) as e:
-            current_app.logger.info.info(
-                'Failed to convert "{0}" to orcid.').format(
-                    sender.get(record_identifier))
+            current_app.logger.info(
+                'Failed to convert "{0}" to orcid.'.format(
+                    sender.get(record_identifier)))
 
 
 def get_author_collection_records_from_valid_authors(authors_refs):
     """Query elasticsearch for the author of the given authors references."""
     es_query = {
-        "filter": {
-            "bool": {
-                "must": [
-                    {"terms": {
-                        "self.$ref": authors_refs
-                    }}, {"match": {
-                        "ids.type": "ORCID"
-                    }}
-                ]
-            }
+        "bool": {
```

Comment:
You can use Elasticsearch-DSL to build the query (otherwise it's like writing raw SQL).